getting-started/Ch-GettingStartedGoDevelopers.rst

```diff
@@ -6,113 +6,117 @@ Get EdgeX Foundry - Go Developers
 Introduction
 ============
 
-These instructions are for Go Lang Developers to obtain and run Go-based EdgeX Foundry microservices. (Users should read: :doc:`../Ch-GettingStartedUsers`)
+These instructions are for Go Lang Developers and Contributors to obtain, run and otherwise work with Go-based EdgeX Foundry microservices. (Users should read: :doc:`../Ch-GettingStartedUsers`)
 
 ================================
 What You Need For Go Development
 ================================
 
-In additional to the hardware and software listed in the Developers page, you will need the following to develop/work with the EdgeX Go-based microservcices.
+In additional to the hardware and software listed in the Developers page, you will need the following to work with the EdgeX Go-based microservcices.
 
-**Go** - The open sourced microservices of EdgeX Foundry are written in Go 1.9. See https://golang.org/dl/ for download and installation instructions.  Newer versions of Go are available and may work, but the project has not thorougly built and tested to these newer versions of the language.  Older versions of Go, especially 1.6 or older, are likely to cause issues.
+**Go**
 
-It is suggested that you create a set of directories for your Go work (including a /bin and /src folder) and set the GOPATH env variable to that directory. This will assist in getting your environment setup and makes it easier to understand how to find code, tools and 3rd party packages.
+The open sourced microservices of EdgeX Foundry are written in Go 1.12. See https://golang.org/dl/ for download and installation instructions.  Newer versions of Go are available and may work (as of this writing - although Go version 1.13 will work as well), but the project has not thorougly built and tested to these newer versions of the language.  Older versions of Go, especially 1.10 or older, are likely to cause issues (EdgeX now uses Go Modules which were introduced with Go Lang 1.11).
 
-**An IDE** - There are many options for writing Go Lang code and one could use a simple text editor.  This guide demonstrates how to get started with JetBrains Go Land.
+It is suggested that you create a set of directories for your Go work (including a /bin and /src folder) and set the GOPATH env variable to that directory. This will assist in getting your environment setup and makes it easier to understand how to find code, tools and 3rd party packages.  Specifically, and as shown below, create a 'go' folder with /src/github.com/edgexfoundry sub-folders.  When you pull (see below) EdgeX code in from its repositories, add the repository folder under /edgexfoundry (in this example, edgex-go has been downloaded and its repository contents would comprise a new subfolder under /src/github.com/edgexfoundry).
```

Comment:
We need to update this section as well. With the advent of modules, the whole idea is that you can place it wherever you like -- specifically and intentionally outside of your gopath to avoid conflicts with solutions that are not using go modules. Additionally, 3rd party modules will not be placed in this directory structure as mentioned -- they will go into a modules directory managed by go. Let me know if you want me to take a stab at rewriting this.

Reply:
@rsdmike - I am taking out the words "suggested" and just leave it as one way to set up your system for Go work is to ....    And I am also adding a comment that with modules, you can place 3rd party modules outiside your gopath to avoid potential conflicts with solutions that are not using go modules (and then refer to an web site like this:  https://medium.com/mindorks/create-projects-independent-of-gopath-using-go-modules-802260cdfb51).  We can then get the PR in and allow you or others to make it more complete/detailed as you see fit.  Make sense?

Reply:
I think so? your 3rd party modules and their location is managed by go, its your project that resides outside of gopath. But yeah, if you wanna link to that article -- thats perfect as it explains it in detail.